CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# This file was generated by https://github.com/audrow/update-ros2-repos
+* @methylDragon @quarkytale @gbiggs
```

Comment:
It looks like github is complaining since these developers don't have write access to our repo, could we remove this?

Reply:
yeah we can